packages/xdl/src/detach/Detach.js

```diff
@@ -381,7 +381,7 @@ async function prepareDetachedServiceContextIosAsync(projectDir: string, args: a
   // path generated inside IosShellApp. When we support more than one path, this needs to
   // be smarter.
   const expoRootDir = path.join(projectDir, '..', '..');
-  const workspaceSourcePath = path.join(projectDir, 'default');
+  const workspaceSourcePath = path.join(projectDir, 'ios');
```

Comment:
Don't we have to do some `if (version >= 33)` here and later on?

Reply:
I believe we don't, this can run only in a service context (CI), that means it's used only when creating a shell app, which are always built with unversioned SDK 🙂